release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1571,22 +1571,101 @@ With this release, availability sets are no longer modified after creation, allo
+
+* Previously, after installing a cluster on {aws-short} with minimum permissions in an existing VPC with a user-provided public IPv4 pool, the cluster could not be destroyed due to a missing permission. With this update, the installation program propagates the `ec2:ReleaseAddress` permission so that the cluster can be destroyed. (link:https://issues.redhat.com/browse/OCPBUGS-49594[OCPBUGS-49594])
+
+* Previously, the installer for {vmw-first} did not validate the number of networks provided in the `install-config.yaml` for failure domains. This caused the installation to proceed with an unsupported configuration if more than the maximum of 10 networks were specified, without providing an error. With this release, the installer now validates the number of configured networks, which resolves the issue by preventing the use of a configuration that exceeds the maximum limit. (link:https://issues.redhat.com/browse/OCPBUGS-49351[OCPBUGS-49351])
+
+* Previously, installing a cluster on {aws-short} with existing subnets (BYO VPC) in Local or Wavelength zones resulted in the edge subnets resource missing the `kubernetes.io/cluster/<InfraID>:shared` tag. With this release, a fix ensures that all subnets used in the `install-config.yaml` file have the required tags. (link:https://issues.redhat.com/browse/OCPBUGS-48827[OCPBUGS-48827])
+
+* Previously, an issue prevented configuring multiple subnets in the failure domain of a Nutanix cluster during installation. The issue is resolved in this release. (link:https://issues.redhat.com/browse/OCPBUGS-49885[OCPBUGS-49885])
+
+* Previously, when installing a cluster on {aws-short}, the `ap-southeast-5` region was not available in the installation program survey, even though this region was supported by {product-title}. With this update, the `ap-southeast-5` region is available. (link:https://issues.redhat.com/browse/OCPBUGS-47681[OCPBUGS-47681])
+
+* Previously, when destroying a cluster installed on {gcp-short}, some resources could be left behind because the installation program did not wait to ensure that all destroy operations completed successfully. With this update, the destroy API waits to ensure that all resources are appropriately deleted. (link:https://issues.redhat.com/browse/OCPBUGS-47489[OCPBUGS-47489])
+
+* Previously, when installing a cluster on {aws-short} in the `us-east-1` regions, the installation could fail if no zone is specified in the `install-config.yaml` file because the `use1-az3` zone does not support any instance types supported by {product-title}. With this update, the installation program prevents the use of the `use1-az3` zone when no zones are specified in the installation configuration file. link:https://issues.redhat.com/browse/OCPBUGS-47477[OCPBUGS-47477]
```

Comment:
```suggestion
* Previously, when installing a cluster on {aws-short} in the `us-east-1` regions, the installation could fail if no zone is specified in the `install-config.yaml` file because the `use1-az3` zone does not support any instance types supported by {product-title}. With this update, the installation program prevents the use of the `use1-az3` zone when no zones are specified in the installation configuration file. (link:https://issues.redhat.com/browse/OCPBUGS-47477[OCPBUGS-47477])
```